beetmoverscript/task.py

```diff
@@ -1,14 +1,19 @@
+import arrow
 import logging
 import os
 import re
+import urllib.parse
 
 from copy import deepcopy
 from scriptworker import client
 
 from beetmoverscript import utils, script
-from beetmoverscript.constants import (STAGE_PLATFORM_MAP,
-                                       RESTRICTED_BUCKET_PATHS,
-                                       CHECKSUMS_CUSTOM_FILE_NAMING)
+
```

Comment:
This blank line is not needed. These are local application/library specific imports and according with the [PEP8 documentation](https://www.python.org/dev/peps/pep-0008/#imports), should be chunked together.